experimental/split_grpc/client/build.rs

```diff
@@ -23,5 +23,6 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             &["../../../examples/hello_world/proto/hello_world.proto"],
             &["../../../examples/hello_world/proto/"],
         )?;
+    println!("cargo:rerun-if-changed=../../../examples/hello_world/proto/hello_world.proto");
```

Comment:
According to https://doc.rust-lang.org/cargo/reference/build-scripts.html#life-cycle-of-a-build-script , 

> The build script will be rebuilt if any of its source files or dependencies change.

So why is this being re-run (before this PR)? AFAICT no source files or dependencies have changed.

Reply:
Good point, I think this is because running the scripts _does_ update the `mtime` of the newly generated `*.rs` file, which then causes cargo to think it's sources have changed. (i.e. it doesn't know that the generated file is an output rather than input.)

`cargo:rerun-if-changed=PATH` is then working (I haven't checked) because its actually removing these generated files from the "watched" path

Reply:
I agree that this is because the generated .rs file is in the source tree.